documentation/docs/configuration/manifest.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_position: 2
+---
+
+# Deployment Manifest
+
+In order for monaco to know what to deploy, there has to be a manifest file present.
+This file provides details on what to deploy and where to deploy it.
+
+## Structure
+
+The manifest is written in yaml syntax. It has two top level keys `projects` and `environments`.
+
+### Projects
+
+All entries under the `projects` top level key specify projects to deploy by monaco. To specify
+the type of a project, one has to provide the `type` key in the project item.
+
+There are currently two types of projects:
+- simple
+- grouping
+
+#### Simple Projects
+This is the default type. All you need to provide is a `name` and `path` property.
+If no `path` property is provided, the name will be used as `path`.
+
+**Note**: It is not allowed for the name to contain either `/` nor `\`. This decision
+was made to explicitly distinquish it from filesystem paths.
+
+**Note**: Paths should always use `/` as separator!
```

Comment:
maybe add a " - no matter if you use Windows, Mac or Linux"

also do we enforce that?

Reply:
Good point!
Yes it is enforced. w